backend/routes/event.js

```diff
@@ -89,16 +89,16 @@ router.delete("/:id", auth, (req, res, next) => {
 // Should we allow updation to events, requires disussion
 
 router.put("/:id", auth, (req, res) => {
-  const event = Event.findOne({ _id: req.params.id })
-    .then(() => {
+  Event.findOne({ _id: req.params.id })
+    .then((event) => {
```

Comment:
Ops. Completely missed it! We need to set up some sort of CI for backend to avoid merging malfunctioned code.

Reply:
Yes, find some way and report here @EmperorYP7 !